pyvista/plotting/themes.py

```diff
@@ -89,8 +93,8 @@ def load_theme(filename):
     Examples
     --------
     >>> import pyvista as pv
-    >>> from pyvista.plotting.themes import DocumentTheme
-    >>> theme = DocumentTheme()
+    >>> from pyvista.plotting.themes import Theme
+    >>> theme = Theme.document_theme()
```

Comment:
`load_theme` currently also accepts a `Theme` object.  This existing before this PR.  This either needs to be documented or split out.